### DISCUSSION TITLE
[BUG] github action 没有运行

### DISCUSSION THREAD

**Discussion by wkk-dev** [dreamhunter2333/cloudflare_temp_email#468] [General]
## 复现步骤
部署失败
Project had an error
![image](https://github.com/user-attachments/assets/e3387272-1ced-43e0-903b-79fc9da2691c)

## 预期行为
完成部署

## 部署方式

- [x]  用户界面部署

## 浏览器环境

Windows11 Edge

##具体问题

github action 没有运行
如图
![image](https://github.com/user-attachments/assets/fe7e968d-aae8-4a39-9748-66f146ed437c)
api权限如下
![image](https://github.com/user-attachments/assets/f81fd488-32ee-4282-b669-def48e7aa864)
![image](https://github.com/user-attachments/assets/ea035548-9460-4443-8957-4ebfd1599f3e)

**Comment by dreamhunter2333** [comment]
感谢反馈，需要手动点击来运行
